common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -477,4 +477,17 @@ public void stopYubiKitManagerUsbDiscovery() {
         }
     }
 
+    /**
+     * A wrapper to telemet results from certificate based authentication (CBA) if CBA occurred.
```

Comment:
I'm not familiar with the use of telemet... is this being used as a verb?  If so, suggest emit telemetry instead.

Reply:
replacing all the weird conjugations of telemetry... telemet, telemeted, telemeterize